### PR TITLE
FE frontend fixs 

### PR DESCRIPTION
**Pull request by kalidiagne**
Related to https://github.com/jubmoji/jubmoji.quest/issues/73 

extra: 

- [x] “Back to app” in /collect doesn’t take you to jubmojis page 
- [x] Update the power’s icon on footer,

### CODE REVIEW

File: apps/jubmoji-quest/src/components/cards/PowerCard.tsx [hunk 1]
```diff
@@ -32,25 +33,41 @@ const PowerCard = ({
...
+                >
                   {description}
                 </Card.Description>
               )}
             </div>
           </div>
+          <button
+            type="button"
+            onClick={(e) => {
+              console.log("button clicked");
```
Comment (AndrewCLu) [inline apps/jubmoji-quest/src/components/cards/PowerCard.tsx:63]: lol

File: apps/jubmoji-quest/src/components/cards/PowerCard.tsx [hunk 2]
```diff
@@ -32,25 +33,41 @@ const PowerCard = ({
...
+              e.stopPropagation();
+              e.nativeEvent.stopImmediatePropagation();
```
Comment (AndrewCLu) [inline apps/jubmoji-quest/src/components/cards/PowerCard.tsx:65]: What do the above two lines do? Can we get a comment

File: apps/jubmoji-quest/src/pages/jubmojis/index.tsx [hunk 3]
```diff
@@ -122,25 +122,23 @@ export default function JubmojisPage() {
       return <Message>No results found.</Message>;
     }
 
-    if (jubmojis.length !== 0)
```
Comment (AndrewCLu) [inline apps/jubmoji-quest/src/pages/jubmojis/index.tsx:125]: Do we have an issue tracking this fix?

**Review by AndrewCLu — APPROVED** [review]
This looks amazing! Just a few nits, but it should be good to go!

File: apps/jubmoji-quest/src/components/cards/PowerCard.tsx (hunk 2 above)
Comment (kalidiagne) [reply apps/jubmoji-quest/src/components/cards/PowerCard.tsx:65]: will do this in other PR, all the code need to bookmark power and quest

File: apps/jubmoji-quest/src/pages/jubmojis/index.tsx (hunk 3 above)
Comment (kalidiagne) [reply apps/jubmoji-quest/src/pages/jubmojis/index.tsx:125]: Open now a issue for this, and will implement when the design is ready. thk!

**Pull request merged**